lib/arInstall.class.php

```diff
@@ -299,6 +299,7 @@ public static function checkSearchConnection($options)
 
         $curlHttpCode = curl_getinfo($curl, CURLINFO_HTTP_CODE);
         curl_close($curl);
```

Comment:
`curl_close()` can be removed as it no longer does anything. See PHP 8 breaking changes [here](https://www.php.net/manual/en/migration80.incompatible.php#migration80.incompatible.resource2object) - search for curl.